### ISSUE TITLE
Print Sequence "one at a time" sequence issues making head bounce/build plate not fully usable

### ISSUE THREAD

**Issue by BBartV** [Ultimaker/Cura#6327]
**Application version**
4.2.1
**Platform**
Win10

**Printer**
UM2+

**Reproduction steps**
Slicing of multiple items to print one at a time is working OK. But compared with verion 4.0.0 and others before the left up corners are not fully usable for my type of projects as I have to move the shapes a bit out of the line .... After that's printing is not working as they are very close and sequence of printing is strange creating an issue as head bounces to another item.

**Actual results**
When printing the printer doesn't start at the front row (the row on the opening side of the printer) but starts with the sencond row of each line making that when the printers goes to the front the head bounces to the item on the second row as this is very close to that row ...due to slicing is no longer working when all items are in a row like before (one olver 3.x version also had that issue !!! But with all older version printing started at the left front corner.

**Expected results**
possiblility to load in the lines like before and print always in front and than fo to the back (in 4.0.0 and bofore this was first the left line, then the middele and than to right always from from to back)

**Additional information**
Hope you can give priority to this a we print quiet a lot and putting less shapes is slowing down a lot !
[sequence-bluid plate issues.zip](https://github.com/Ultimaker/Cura/files/3594354/sequence-bluid.plate.issues.zip)

**Comment by Ghostkeeper** [comment]
It's a bit hard to follow. Let me summarise:
* The left front corner is not fully usable on the build plate. (Perhaps it's hitting the metal clips on the build plate?)
* The head bounces (collides?) with previously printed objects.
* The printing order is wrong, which causes the print head to collide.

Looking at the shape of the print head collision area, it looks to be about correct. It leaves 13mm clearance front-to-back. I'm measuring it with calipers at 9mm but 4mm extra clearance for safety isn't a bad idea. It leaves 43mm clearance side-to-side. I'm measuring 40mm there, so there's 3mm extra margin. Seems okay to me.

But indeed the ordering seems to be wrong. If I place 3 cubes in a line along the Y axis, I get it printing from the back to the front. The 9mm I measured is on the front side which is the shortest side for the UM2, so it should be printing front-to-back.

**Comment by GregValiant** [comment]
I'm cleaning house.
Is this still a problem with current Cura versions (5.8.0 and up)?  Can I close this?
